src/components/BasicMap/BasicMap.js

```diff
@@ -216,6 +216,17 @@ class BasicMap extends Component {
     }
   }
 
+  componentWillUnmount() {
+    const { onFeaturesHover } = this.props;
+    ['moveend', 'singleclick'].forEach(event => {
+      this.map.removeEventListener(event);
+    });
+
+    if (onFeaturesHover) {
+      this.map.removeEventListener('pointermove');
```

Comment:
Does it work ?   I suggest to use `unByKey`  instead https://github.com/geops/react-spatial/blob/50bf6a28202db4065678745976a6aa4e37a4c429/src/components/Popup/Popup.js

Reply:
Indeed, good method to un-register ol event listener (so not intuitive name though :anguished: )